website/content/en/v0.37/getting-started/migrating-from-cas/_index.md

```diff
@@ -92,10 +92,10 @@ One for your Karpenter node role and one for your existing node group.
 First set the Karpenter release you want to deploy.
 
 ```bash
-export KARPENTER_VERSION=v0.34.6
+export KARPENTER_VERSION="0.37.0"
```

Comment:
Is this right?

Reply:
I meant specifically about the lack of the v

Reply:
Ah got it. Yep, this is correct. We dropped the v from the image tag at 0.35.0 for better semver compliance. It's still used in links but in the getting started guide it should be dropped.